pkg/diagnostics/v2/client.go

```diff
@@ -0,0 +1,43 @@
+package v2
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/dcos/dcos-diagnostics/api/rest"
+)
+
+// Client is a REST API wrapper around the new Diagnostics API.
+type Client struct {
+	http *httpclient.Client
+}
+
+// NewClient creates a new Client.
+func NewClient(baseClient *httpclient.Client) *Client {
+	return &Client{
+		http: baseClient,
+	}
+}
+
+// List gets a list of all cluster bundles.
+func (c *Client) List() ([]rest.Bundle, error) {
+	resp, err := c.http.Get("/system/health/v1/diagnostics")
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	switch {
```

Comment:
Can we go for our standard pattern or does diagnostics return something else than a `200` in some cases?
```
switch resp.StatusCode {
case 200:
    ...
default:
    return nil, fmt.Errorf("unexpected status code %d", resp.StatusCode)
}